pipeline-modules/infra-service/azure/terraform.yaml

```diff
@@ -206,4 +206,5 @@ template: |
       {{ upload_artifact(tf_state_file) }}  {# upload the state even if the TF action failed as it may have already been modified #}
       {% endif %}
       az logout
+    timeoutInMinutes: 300
```

Comment:
what is this timeout for? Terraform apply/destroy operations can easily take longer than 5 minutes....

Reply:
The timeout is in minutes to 300 mins.